libs/shared/ui/src/lib/components/navbar/navbar.tsx

```diff
@@ -1,13 +1,31 @@
 import { Link } from 'react-router-dom'
-import { INDEX_URL } from '@console/shared/utils'
+import { OVERVIEW_URL } from '@console/shared/utils'
+interface NavbarProps {
+  className?: string
+  logoUrl?: string
+  progress?: number
+  contentLeft?: React.ReactElement
+}
+
+export function Navbar(props: NavbarProps) {
+  const { className, progress = 0, logoUrl = OVERVIEW_URL, contentLeft } = props
 
-export function Navbar() {
   return (
-    <nav className="flex items-center h-16 w-23 bg-element-light-darker-300">
-      <div className="h-full">
-        <Link to={INDEX_URL} className="flex items-center px-6 border-r border-element-light-darker-100 h-full">
-          <img className="w-[90px]" src="/assets/logos/logo-white.svg" alt="Qovery logo white" />
-        </Link>
+    <nav
+      className={`flex items-center h-16 w-23 bg-white border-b border-element-light-lighter-400 mb-[6px] ${
+        className || ''
+      }`}
+    >
+      <div className="relative h-full w-full">
+        <div className="h-full flex items-center">
+          <Link to={logoUrl} className="flex items-center px-6 border-r border-element-light-lighter-400 h-full">
+            <img className="w-[90px]" src="/assets/logos/logo-black.svg" alt="Qovery logo black" />
+          </Link>
+          {contentLeft && contentLeft}
```

Comment:
usage of this condition ?

Reply:
Maybe some react magic we are not aware of? I have to confess I don't really see the usage here 😮